Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h

```diff
@@ -0,0 +1,87 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// @file   AODProducerWorkflowSpec.h
+
+#ifndef O2_AODPRODUCER_WORKFLOW_SPEC
+#define O2_AODPRODUCER_WORKFLOW_SPEC
+
+#include "DataFormatsFT0/RecPoints.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/DataProcessorSpec.h"
+#include "Framework/Task.h"
+#include "ReconstructionDataFormats/TrackTPCITS.h"
+#include "TStopwatch.h"
+#include <string>
+#include <vector>
+
+using namespace o2::framework;
+
+namespace o2
+{
+namespace aodproducer
+{
+
+using TracksTable = o2::soa::Table<o2::aod::track::CollisionId,
+                                   o2::aod::track::TrackType,
+                                   o2::aod::track::X,
+                                   o2::aod::track::Alpha,
+                                   o2::aod::track::Y,
+                                   o2::aod::track::Z,
+                                   o2::aod::track::Snp,
+                                   o2::aod::track::Tgl,
+                                   o2::aod::track::Signed1Pt,
+                                   o2::aod::track::SigmaY,
+                                   o2::aod::track::SigmaZ,
+                                   o2::aod::track::SigmaSnp,
+                                   o2::aod::track::SigmaTgl,
+                                   o2::aod::track::Sigma1Pt,
+                                   o2::aod::track::RhoZY,
+                                   o2::aod::track::RhoSnpY,
+                                   o2::aod::track::RhoSnpZ,
+                                   o2::aod::track::RhoTglY,
+                                   o2::aod::track::RhoTglZ,
+                                   o2::aod::track::RhoTglSnp,
+                                   o2::aod::track::Rho1PtY,
+                                   o2::aod::track::Rho1PtZ,
+                                   o2::aod::track::Rho1PtSnp,
+                                   o2::aod::track::Rho1PtTgl>;
```

Comment:
shouldn't this be defined as part of the data model (or put into if not present)?

Reply:
@jgrosseo @aalkin : Could you comment on this?

Reply:
This is true, `StoredTracks` and `StoredTracksCov` cannot be joined 'as is'. This definition should be put in data model, as a workaround for now, then I'll make it properly depend on existing definitions.

Reply:
We can use defines not to repeat the list of columns for each case, but it is not so nice.